metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/StormStatusServiceImpl.java

```diff
@@ -17,10 +17,7 @@
  */
 package org.apache.metron.rest.service.impl;
 
-import org.apache.metron.rest.model.TopologyResponse;
-import org.apache.metron.rest.model.TopologyStatus;
-import org.apache.metron.rest.model.TopologyStatusCode;
-import org.apache.metron.rest.model.TopologySummary;
+import org.apache.metron.rest.model.*;
```

Comment:
@merrimanr @mattf-horton We did not have an official vote.  Given the widespread acceptance of it, since it's not a dev guildelines and not a bylaw thing, my impression was we didn't need it. Having said that, if it needs or should have an actual vote, I'm definitely okay with it and it should be done before https://github.com/apache/metron/pull/577 goes in.

However, 577 is still outstanding pending additional feedback I requested.  I was inclined to not update the wiki, because we hadn't built out and validated the tooling at the time.  I personally lean towards updating the wiki once we're good with that PR going in.